CHANGELOG.next.toml

```diff
@@ -10,6 +10,18 @@
 # references = ["smithy-rs#920"]
 # meta = { "breaking" = false, "tada" = false, "bug" = false }
 # author = "rcoh"
+[[aws-sdk-rust]]
+message = "If you directly depend on AWS or Smithy runtime crates _(e.g., AWS crates not named aws-config or prefixed with aws-sdk-)_, the default features from those crates must now be explicitly set in your `Cargo.toml`."
```

Comment:
```suggestion
message = "If you directly depend on AWS or Smithy runtime crates _(e.g., AWS crates not named `aws-config` or prefixed with `aws-sdk-`)_, the previously default features from those crates must now be explicitly set in your `Cargo.toml`."
```

Reply:
Thinking we should probably list out the previous defaults for each runtime crate as a quick upgrade reference here.